www/grpc/network.go

```diff
@@ -48,12 +48,16 @@ func (s *networkServer) GetNodeInfo(_ context.Context,
 }
 
 func (s *networkServer) GetNetworkInfo(_ context.Context,
-	_ *pactus.GetNetworkInfoRequest,
+	req *pactus.GetNetworkInfoRequest,
 ) (*pactus.GetNetworkInfoResponse, error) {
 	ps := s.sync.PeerSet()
 	peerInfos := make([]*pactus.PeerInfo, 0, ps.Len())
 
 	ps.IteratePeers(func(peer *peerset.Peer) bool {
+		if req.OnlyOnline && (!peer.IsConnected() || !peer.IsKnown()) {
```

Comment:
```suggestion
		if req.OnlyOnline && !peer.IsConnected() {
```

`IsConnected()` checks for both `Connected` and `Known` status